runelite-client/src/main/java/net/runelite/client/plugins/statusbars/StatusBarsPlugin.java

```diff
@@ -47,10 +59,58 @@
 	@Inject
 	private OverlayManager overlayManager;
 
+	@Inject
+	private Client client;
+
+	@Inject
+	private StatusBarsConfig config;
+
+	@Getter(AccessLevel.PACKAGE)
+	private Instant lastCombatAction;
+
 	@Override
 	protected void startUp() throws Exception
 	{
-		overlayManager.add(overlay);
+	}
+
+	void updateLastCombatAction()
+	{
+		this.lastCombatAction = Instant.now();
+	}
+
+	@Subscribe
+	public void onGameTick(GameTick gameTick)
+	{
+		if (!config.toggleRestorationBars())
+		{
+			overlayManager.add(overlay);
+			return;
+		}
+		else
+		{
+			final Actor interacting = client.getLocalPlayer().getInteracting();
+			final boolean isNpc = interacting instanceof NPC;
+			final int COMBAT_TIMEOUT = config.hideStatusBarDelay();
```

Comment:
```suggestion
			final int combatTimeout = config.hideStatusBarDelay();
```

The value is not a constant value, thus it shouldn't be written as one.